build/localnet/manifests/cluster-manager.yaml

```diff
@@ -21,4 +21,41 @@ spec:
       env:
         - name: RPC_HOST
           value: pocket-full-nodes
-  serviceAccountName: cluster-manager-account
+  serviceAccountName: cluster-manager
```

Comment:
Was going to ask you questions but ChatGPT explained everything :)

![ChatGPT-Explaining_Kubernetes_Manifest_Components](https://github.com/pokt-network/pocket/assets/1892194/ec6f8c3c-233a-406b-a422-4172c3d619f9)